Coop_Listing_Site/Coop_Listing_Site/Web.config

```diff
@@ -15,10 +15,10 @@
   </connectionStrings>
 
   <!-- Force SSL/TLS the first time the site is visited, and for all subsequent visits for the next year using HTTP Strict Transport Security (HSTS) -->
-  <!-- Uncomment when deploying to the web server
+  <!-- Uncomment when deploying to the web server-->

```

Comment:
@droopy25 Why did we have this commented out in the first place again? I know the 

> when deploying to the web server...part
but was there a reason we were waiting?